extensions/wikia/EmbeddableDiscussions/scripts/ext.embeddableDiscussions.js

```diff
@@ -197,10 +197,12 @@ require([
 
 	$(function () {
 		$('.embeddable-discussions-module').on('click', '.upvote', function (event) {
-			var upvoteUrl = getBaseUrl() + event.currentTarget.getAttribute('href'),
+			var postId = event.currentTarget.getAttribute('data-id'),
 				hasUpvoted = event.currentTarget.getAttribute('data-hasUpvoted') === '1',
-				$svg = $($(event.currentTarget).children()[0]),
-				verb = hasUpvoted ? 'DELETE' : 'POST';
+				method = hasUpvoted ? 'downVotePost' : 'upVotePost',
+				upvoteUrl = mw.config.get('wgScriptPath') + 
+					'/wikia.php?controller=DiscussionVote&method=' + method + '&postId=' + postId,
+				$svg = $($(event.currentTarget).children()[0]);
```

Comment:
Is there a reason why you define variables:
1. with `var`? (less important)
2. using comma? (more important)

Reply:
because that is how we did it in the app js files. I did not want to introduce new conventions just in this single place

Reply:
(const and let are not supported in these js files afair)